src/styles/box/header.scss

```diff
@@ -0,0 +1,102 @@
+.header {
+  position: relative;
+  width: 1024px;
```

Comment:
Using a fixed width of 1024px for the header may cause layout issues on smaller screens. Consider using a percentage or max-width for better responsiveness.